egs/fisher_callhome_spanish/st1/run.sh

```diff
@@ -123,10 +95,9 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
 
         steps/make_fbank_pitch.sh --cmd "$train_cmd" --nj 32 --write_utt2num_frames true \
             data/${x} exp/make_fbank/${x} ${fbankdir}
-        utils/fix_data_dir.sh data/${x}
     done
 
-    # speed-perturbed
+    # speed-perturbed. data/${train_set_ori} is the orignal and data/${train_set} is the augmented
```

Comment:
you can remove `fisher_train` from previous `steps/make_fbank_pitch.sh` loop because you finally use speed perturbed fbank `train_sp`

Reply:
I can't understand this because fisher_train is necessary to generate train_sp.

Reply:
Also, it's dangerous to skip this for future research because `fisher_train/feats.scp` is missed.

Reply:
https://github.com/espnet/espnet/blob/497acb0cbc12f0f2803599cdef5917a4eb0f8989/egs/csj/asr1/run.sh#L88-L102

see CSJ recipe for example. you can create `--use-speed-perturb` option and apply sp like this
```bash

use_speed_perturb=true

...

if $use_speed_perturb; then
    train_set=${train_set_ori}
else
    train_set=${train_set_ori}_sp
fi
...

if $use_speed_perturb; then
    # speed purturbation
    utils/perturb_data_dir_speed.sh 0.9 data/${train_set_ori} data/temp1
    utils/perturb_data_dir_speed.sh 1.0 data/${train_set_ori} data/temp2
    utils/perturb_data_dir_speed.sh 1.1 data/${train_set_ori} data/temp3
    utils/combine_data.sh --extra-files utt2uniq data/${train_set} data/temp1 data/temp2 data/temp3
    rm -r data/temp1 data/temp2 data/temp3
fi

    # feature extraction
    fbankdir=fbank
    for x in ${train_set} ${train_dev}; do
        steps/make_fbank_pitch.sh --cmd "$train_cmd" --nj 32 --write_utt2num_frames true \
            data/${x} exp/make_fbank/${x} ${fbankdir}
        utils/fix_data_dir.sh data/${x}
        utils/validate_data_dir.sh data/${x}
    done
```

Reply:
@sw005320 Yes, I adopted this design at first, but @sw005320 said it's complicated.
So someone should change CSJ recipe.

Reply:
Oh, I'm sorry. CSJ seemed good.